cpp/src/barretenberg/stdlib/hash/sha256/sha256.test.cpp

```diff
@@ -169,15 +179,15 @@ TEST(stdlib_sha256, test_duplicate_proving_key)
 
 TEST(stdlib_sha256, test_plookup_55_bytes)
 {
-    typedef stdlib::field_t<plonk::UltraComposer> field_pt;
-    typedef stdlib::packed_byte_array<plonk::UltraComposer> packed_byte_array_pt;
+    typedef plonk::stdlib::field_t<plonk::UltraComposer> field_pt;
+    typedef plonk::stdlib::packed_byte_array<plonk::UltraComposer> packed_byte_array_pt;
 
     // 55 bytes is the largest number of bytes that can be hashed in a single block,
     // accounting for the single padding bit, and the 64 size bits required by the SHA-256 standard.
-    plonk::UltraComposer composer = UltraComposer();
+    plonk::UltraComposer composer = plonk::UltraComposer();
```

Comment:
Could consider just `plonk::UltraComposer composer;` or `auto composer = plonk::UltraComposer();`. Doesn't really matter, just good to be consistent across bberg.

Reply:
Thanks, I agree. I went with `auto` because that's what we use in the stdlib primitives tests.